website/src/components/Features.tsx

```diff
@@ -1,26 +1,27 @@
-import Image from 'next/image'
-
+import { useTranslation } from 'react-i18next';
 import { CheckIcon } from './CheckIcon'
+import Image from 'next/image'
 import sampleImage from '@/images/sampleImage.png'
 
-const features = [
-  'Select the news topics you want to follow',
-  'Choose when you\'d like to receive your news: daily, weekly, or on Fridays.',
-  'Select the sources to create your personalized newsletter.',
-]
-
 export function Features() {
+  const { t } = useTranslation();
+
+  const features = [
+    t('feature3'),
+    t('feature4'),
+    t('feature5')
+  ]
```

Comment:
```suggestion
```
You don't use it, no need to create an array.

Reply:
We are using it from line 27 to 32 :
`{features.map((feature) => (
                <li key={feature} className="flex">
                  <CheckIcon className="h-8 w-8 flex-none fill-cyan-700" />
                  <span className="ml-4">{feature}</span>
                </li>
              ))}`